src/Containers/App/index.jsx

```diff
@@ -267,13 +269,57 @@ class App extends Component {
       const target = escs.targets[i];
       const esc = escs.individual.find((esc) => esc.index === target);
 
+      const info = await this.serial.getFourWayInterfaceInfo();
```

Comment:
I am not happy about this block here at all. This should not be done in the Conatainer, rather before flashing in `FourWay.js`

Reply:
i moved it to a `flashPreflight` method inside `FourWay.js`